apps/sms/views/conversation/test/unit/link_helper_test.js

```diff
@@ -192,6 +192,11 @@ suite('link_helper_test.js', function() {
         testURLOK('http://whatever.stuff');
         testURLOK('https://whatever.stuff');
       });
+
+      test('Bug 941763', function() {
+        testURLNOK('.website.com');
```

Comment:
I think we wanted to link the part after `.` ? Maybe add `.` to the safeStart regexp ?